src/gurt/tests/test_gurt.c

```diff
@@ -2050,6 +2049,7 @@ main(int argc, char **argv)
 
 	rc =  cmocka_run_group_tests(tests, init_tests, fini_tests);
 	if (rc != 0) {
```

Comment:
don't need '{}' here

Reply:
This code has been verified with checkpatch before commit. And it was no warning about line 2052.
I don't think the style warning do node use {} for a single line is right, but I'm not argue with the coding standard.
If {} covers more than one line it's legal. Do you really suggest to add a comment for each case like that?